label_studio/core/views.py

```diff
@@ -208,6 +216,30 @@ def localfiles_data(request):
     return HttpResponseForbidden()
 
 
+def aperturedb_data(request):
+    """Serving files for ApertureDBImportStorage"""
+    user = request.user
+    title = request.GET.get('title')
+    uid = request.GET.get('key')
+    if title and uid and user.is_authenticated:
+        for storage in ApertureDBImportStorage.objects.all():
```

Comment:
it doesn't look like other storages do it this way, is this because aperrturedb both serves and stores?

Reply:
We arrange for the images to be served from ApertureDB via specially configured routes in the LabelStudio app.